.github/workflows/test-libmaxminddb.yml

```diff
@@ -44,36 +44,19 @@ jobs:
         run: sudo apt install libmaxminddb-dev
         if: matrix.platform == 'ubuntu-latest'
 
+      - name: "Work around macos arm64 homebrew directory changes"
+        if: runner.os == 'macos' and runner.arch = 'arm64'
```

Comment:
This might have to be `macOS`

Reply:
It seems like it worked. :shrug:

Reply:
Yeah I'm confused. It should be
```
if: runner.os == 'macOS' && runner.arch == 'ARM64'
```
See Annotations from https://github.com/maxmind/MaxMind-DB-Reader-python/actions/runs/8897863643
```
[Invalid workflow file: .github/workflows/test-libmaxminddb.yml#L48](https://github.com/maxmind/MaxMind-DB-Reader-python/actions/runs/8897863643/workflow)
The workflow is not valid. .github/workflows/test-libmaxminddb.yml (Line: 48, Col: 13): Unexpected symbol: 'and'. Located at position 22 within expression: runner.os == 'macos' and runner.arch = 'arm64'
```

Reply:
Oh, I would have expected some indication that failed in the build statuses or via email. I'll fix that up.